beacon_node/network/src/network_beacon_processor/gossip_methods.rs

```diff
@@ -599,6 +601,75 @@ impl<T: BeaconChainTypes> NetworkBeaconProcessor<T> {
         }
     }
 
+    pub async fn process_gossip_blob_column_sidecar(
+        self: &Arc<Self>,
+        message_id: MessageId,
+        peer_id: PeerId,
+        _peer_client: Client,
+        subnet_id: BlobColumnSubnetId,
+        column_sidecar: Arc<BlobColumnSidecar<T::EthSpec>>,
+        seen_duration: Duration,
+    ) {
+        let slot = column_sidecar.slot();
+        let root = column_sidecar.block_root();
+        let index = column_sidecar.index;
+        let delay = get_slot_delay_ms(seen_duration, slot, &self.chain.slot_clock);
+        // Log metrics to track delay from other nodes on the network.
+        metrics::observe_duration(
+            &metrics::BEACON_BLOB_COLUMN_GOSSIP_SLOT_START_DELAY_TIME,
+            delay,
+        );
+        metrics::set_gauge(
+            &metrics::BEACON_BLOB_COLUMN_LAST_DELAY,
+            delay.as_millis() as i64,
+        );
+        match self
+            .chain
+            .verify_blob_column_sidecar_for_gossip(column_sidecar, *subnet_id)
+        {
+            Ok(gossip_verified_blob_column) => {
+                metrics::inc_counter(
+                    &metrics::BEACON_PROCESSOR_GOSSIP_BLOB_COLUMN_SIDECAR_VERIFIED_TOTAL,
+                );
+
+                debug!(
+                    self.log,
+                    "Successfully verified gossip blob column sidecar";
+                    "slot" => %slot,
+                    "root" => %root,
+                    "index" => %index,
+                );
+
+                self.propagate_validation_result(message_id, peer_id, MessageAcceptance::Accept);
+
+                // Log metrics to keep track of propagation delay times.
+                if let Some(duration) = SystemTime::now()
+                    .duration_since(UNIX_EPOCH)
+                    .ok()
+                    .and_then(|now| now.checked_sub(seen_duration))
+                {
+                    metrics::observe_duration(
+                        &metrics::BEACON_BLOB_COLUMN_GOSSIP_PROPAGATION_VERIFICATION_DELAY_TIME,
+                        duration,
+                    );
+                }
+                self.process_gossip_verified_blob_column(
+                    peer_id,
+                    gossip_verified_blob_column,
+                    seen_duration,
+                )
+                .await
+            }
+            Err(err) => {
+                error!(
+                    self.log,
+                    "Internal error when verifying blob column sidecar";
+                    "error" => ?err,
+                )
```

Comment:
Should propagate a general ignore validation result here, or still early PoC?

Reply:
Yep still early PoC! Plan is to gradually and logically get changes onto the `das` branch.